kratos/input_output/logger_output.cpp

```diff
@@ -53,7 +56,12 @@ namespace Kratos
             switch (message_severity)
             {
             case LoggerMessage::Severity::WARNING:
-                if (mOptions.Is(WARNING_PREFIX)) mrStream << "[WARNING] ";
+                if (mOptions.Is(WARNING_PREFIX)) {
+                    #if defined(KRATOS_COLORED_LOGGING)
```

Comment:
thinking abt it again, wouldn't it make sense to add this globally in the `color_utilities`?
then it would only have to be done once?

Reply:
The `color_utilities` are used in other places as well. Including it there would change the behaviour of existing code. If that is ok it would be cleaner i think. 

Then e.g. the "KYEL" macro could simply be empty (as it is e.g. for Windows in the color_utilities already now.

Reply:
Previously we did not have a consensus on how to include the colors, now we have
It is in core 

what do you think @loumalouomega ? I think you are the only one using it for now

Reply:
I briefly checked, the only place it is used are some convergence criteria.